superbench/analyzer/file_handler.py

```diff
@@ -221,3 +222,29 @@ def output_lines_in_html(lines, output_path):
             f.writelines(html_str)
     except Exception as e:
         logger.error('FileHandler: html_data_output - {}'.format(str(e)))
+
+
+def merge_column_in_excel(ws, row, column):
+    """Merge cells in the selected index of column with continuous same contents.
+
+    Args:
+        ws (worksheet): the worksheet of the excel to process
+        row (int): the max row index to merge
+        column (int): the index of the column to merge
+    """
+    dict_from = {}
+    aligncenter = Alignment(horizontal='center', vertical='center')
+    # record continuous row index (start, end) with the same content
+    for row_index in range(1, row + 1):
```

Comment:
Does worksheet always have the title row?

Reply:
no, it doesn't have title, 1 is the first index in worksheet